pallets/api/src/fungibles/tests.rs

```diff
@@ -47,13 +47,17 @@ mod encoding_read_result {
 
 	#[test]
 	fn token_name() {
-		let name = vec![42, 42, 42, 42, 42];
+		let mut name = Some(vec![42, 42, 42, 42, 42]);
```

Comment:
Do we truly need a variable? It is not reused anywhere except the next line.

Reply:
Yes I think it is personally cleaner.